chore: update document write guide (#119)
bytebase/bytebase.com: docs/document-write-guide.md

6. Run `pnpm dev` start the review server.
7. Go to [localhost:3000/docs](http://localhost:3000/docs) and check your creation.
8. If everything is fine, create a new PR with the changes to [our repo in GitHub](https://github.com/bytebase/bytebase.com).

## Using powerful components into Markdown

> For show more styled block into documention, we support using some powerful components:

Reviewer: ```suggestion
To display more styled blocks in documentation, we support some powerful components:
```

#### Usage

```markdown
<hint-block type="info">

Bytebase has already prepared some sample data. In particular, it has created a Test environment and a Prod environment, each containing a mysql instance. To establish the connection to those instances, one quick way is to [start a MySQL docker instance](#start-a-mysql-docker-instance-for-testing).

</hint-block>
```

#### Props

| Name | Type                | Default | Description        |
| ---- | ------------------- | ------- | ------------------ |
| type | `info` \| `warning` | `info`  | Set the hint style |

### `doc-link-block`

A pretty link block to another page.

![doc-link-block-example](/static/docs-assets/doc-link-block-example.png)

#### Usage

```markdown
<doc-link-block url="/docs/use-bytebase/webhook-integration/overview" title="Webhook Integration"></doc-link-block>
```

#### Props

| Name  | Type     | Default | Description                   |
| ----- | -------- | ------- | ----------------------------- |
| url   | `string` | `''`    | Set the link url              |
| title | `string` | `''`    | Description of the link title |

### `include-block`

> Render other markdown directly.

Reviewer: ```suggestion
Include another markdown and render inline.
```
